projects/core/src/occ/adapters/cms/occ-cms-page.adapter.ts

```diff
@@ -44,16 +49,34 @@ export class OccCmsPageAdapter implements CmsPageAdapter {
    */
   load(pageContext: PageContext): Observable<CmsStructureModel> {
     const params = this.getPagesRequestParams(pageContext);
+    if (this.featureConfigService.isEnabled('newCmsEndpoint')) {
```

Comment:
What do you think about assigning the feature name 'newCmsEndpoint' to a constant export value and relocating it to 'core/src/model'. This way, we can reuse the variable throughout the codebase instead of defining the string each time.

Also I am uncertain if the name 'newCmsEndpoint' is fitting here. Word 'new' is from the time-being perspective risky to use (right now it's new but we don't know the future). Maybe we can switch to:

Example:
```
export const USER_CMS_ENDPOINTS = 'userCmsEndpoints';
```

Reply:
good point